pkgs/misc/ptouch-print/default.nix

```diff
@@ -1,33 +1,47 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchgit
-, autoreconfHook
+, cmake
+, gettext
 , gd
 , libusb1
+, pkg-config
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "ptouch-print";
-  version = "1.4.3";
+  version = "1.5";
 
   src = fetchgit {
-    url = "https://mockmoon-cybernetics.ch/cgi/cgit/linux/ptouch-print.git";
-    rev = "v${version}";
-    sha256 = "0i57asg2hj1nfwy5lcb0vhrpvb9dqfhf81vh4i929h1kiqhlw2hx";
+    url = "https://git.familie-radermacher.ch/linux/ptouch-print.git";
+    rev = "e3c0073466ed99dbde9bbbcceea1c54f64967fc8";
```

Comment:
I was unable to clone tag via `rev = "v1.5"`:

```
these 2 derivations will be built:
  /nix/store/hd9dc4fmg830rrlbcq9irw7pw7m5p6n8-ptouch-print.drv
  /nix/store/2gslij5frlrvk614mwxiyy10y101r79g-ptouch-print-1.5.drv
building '/nix/store/hd9dc4fmg830rrlbcq9irw7pw7m5p6n8-ptouch-print.drv'...
exporting https://git.familie-radermacher.ch/linux/ptouch-print.git (rev v1.5) into /nix/store/nnw8rnnpdijadkxm59d5bxphbfywi2nj-ptouch-print
Initialized empty Git repository in /nix/store/nnw8rnnpdijadkxm59d5bxphbfywi2nj-ptouch-print/.git/
fatal: dumb http transport does not support shallow capabilities
fatal: dumb http transport does not support shallow capabilities
From https://git.familie-radermacher.ch/linux/ptouch-print
 * [new branch]      master     -> origin/master
 * [new tag]         v1.5       -> v1.5
Unrecognized git object type: tag
Unable to checkout refs/tags/v1.5 from https://git.familie-radermacher.ch/linux/ptouch-print.git.
```

not sure what that's about.